### ISSUE TITLE
Can't run static binary

### ISSUE THREAD

**Issue by jtremblay** [torognes/vsearch#556]
Dear vsearch devs,
When I try to run the static binary on a centos 7 system, I get the following error:

```
vsearch v2.26.1_linux_x86_64, 125.5GB RAM, 32 cores
https://github.com/torognes/vsearch

segmentation error (core dumped)

```
Does the same thing with 2.27 and a couple of other earlier versions.
Cheers,

**Comment by torognes** [comment]
Thanks for reporting this problem. I was able to reproduce it by running CentOS 7 in a Docker container.

Static builds do not always work well, and in this case there might be differences between CentOS 7 and Debian 12, on which the latest builds were made, that caused the problem. In general, static builds are not recommended.

I understand your need to run the static binary because the dynamically loaded version requires newer libraries than available on CentOS 7, since CentOS is rather old and soon not supported anymore.

I have now made a build of vsearch 2.27.0 on CentOS 7 with dynamically loaded libraries that should work for you. It is available in the latest release https://github.com/torognes/vsearch/releases/tag/v2.27.0 in the file `vsearch-2.27.0-linux-x86_64-centos7.tar.gz` (https://github.com/torognes/vsearch/releases/download/v2.27.0/vsearch-2.27.0-linux-x86_64-centos7.tar.gz).

In the future we should perhaps consider building the binaries on an older Linux distribution to make them compatible a wider range of systems.
